frontend/src/views/WheresMyGene/components/GetStarted/style.ts

```diff
@@ -43,9 +72,7 @@ export const ColumnOne = styled.div`
 `;
 
 export const ColumnTwo = styled.div`
-  flex: 1 2 calc(10vw);
-
-  z-index: 10;
```

Comment:
removing z-index in ColumnTwo of GetStarted so that users can interact (delete, view gene info) with the gene labels. Previously the z-index was blocking any sort of interactions